src/community/china/_os-settings.md

```diff
@@ -0,0 +1,106 @@
+{% assign id =  include.os | downcase -%}
+
+{% if id == 'windows' -%}
+{% assign shell = 'Powershell' -%}
+{% assign prompt = 'C:\>' -%}
+{% assign envvarset = 'C:\> $env:' -%}
+{% assign setpath = 'C:\> $env:PATH = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"' -%}
+{% assign comtoset = '$env:' -%}
+{% assign unzip='Extract-Archive' %}
```

Comment:
To be consistent with the other commands and fix syntax highlighting, should this start with a `C:\>`?

```suggestion
{% assign unzip='C:\> Extract-Archive' %}
```